client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/submissionFormValidate.js

```diff
@@ -0,0 +1,47 @@
+import { defineMessages } from 'react-intl';
+import { questionTypes } from '../../constants';
+
+const translations = defineMessages({
+  pleaseRecordYourVoice: {
+    id: 'course.assessment.submission.pages.SubmissionEditIndex.submissionFormValidate.pleaseRecordYourVoice',
+    defaultMessage: 'Please record your voice',
+  },
+});
+
+const validateDefaultQuestion = () => undefined;
```

Comment:
Prefer to preserve `undefined` to those variables that are really undefined. perhaps we just return `true` by default?